docker/Dockerfile-multi-arch

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.18.3 as build
+WORKDIR /workspace
+ENV GO111MODULE=on \
+    GOPROXY=https://goproxy.cn,direct \
+    BINARY=postgres-operator
+COPY . .
+RUN CGO_ENABLED=0 go build -o build/linux/${BINARY} ./cmd/main.go
+
+FROM {base_image}
```

Comment:
If someone clones, the project won't have {base_image} defined and will need help figuring out where it comes from. If you use, ARG will help with understanding.
Instead of  sed -i '' 's/{base_image}/${base_arm_image}/g' ./docker/Dockerfile-multi-arch --
use build-args from build-push-action

Reply:
Thanks for the tip.